src/WeavedInterface.php

```
@@ -4,6 +4,13 @@
 
 namespace Ray\Aop;
 
+/** @psalm-import-type MethodBindings from Types */
 interface WeavedInterface
 {
+    /**
+     * @param MethodBindings $bindings
+     *
+     * @SuppressWarnings(PHPMD.CamelCaseMethodName)
```

PHPStan (2.0.3)

PHPDoc tag @SuppressWarnings has invalid value ((PHPMD.CamelCaseMethodName)): Unexpected token ".CamelCaseMethodName)", expected ')' at offset 80 on line 4


(phpDoc.parseError)